### PR TITLE
Refactor a few more element modifiers away

### PR DESCRIPTION
**Pull request by jrjohnson**
Refs ilios/ilios#5374

### CODE REVIEW

File: packages/frontend/app/components/global-search.js [hunk 1]
```diff
@@ -108,13 +123,4 @@ export default class GlobalSearchComponent extends Component {
     this.args.onSelectPage(1);
     this.args.setIgnoredSchoolIds(ignoredSchoolIds);
   }
-
-  search = restartableTask(async (el, [query]) => {
```
Comment (stopfstedt) [inline packages/frontend/app/components/global-search.js:112]: this is incomplete. the component's template is supposed to renders a spinner while the search task is running.

image

it now displays `No results` instead while search is running. 

image

**Review by stopfstedt — CHANGES_REQUESTED** [review]
apart from the missing spinner on the global search, this looks good to me.

File: packages/frontend/app/components/global-search.js (hunk 1 above)
Comment (jrjohnson) [reply packages/frontend/app/components/global-search.js:112]: Fixed, added a test.

**Review by stopfstedt — APPROVED** [review]
LGTM

**Pull request merged**